azure-pipelines.yml

```diff
@@ -49,3 +49,77 @@ jobs:
       testRunTitle: 'Publish test results for $(Agent.JobName) $(TEST_MODE) $(PYTHON_VERSION)'
       failTaskOnFailedTests: true
     condition: succeededOrFailed()
+
+- job:
+  displayName: Linux
+  pool:
+    vmImage: 'ubuntu-18.04'
+  strategy:
+    matrix:
+      ubuntu_python36:
+        DISTRIB: 'ubuntu'
+        PYTHON_VERSION: '3.6'
+        SPHINXOPTS: ''
+      conda_python36_sphinx183:
+        DISTRIB: 'conda'
+        PYTHON_VERSION: '3.6'
+        SPHINX_VERSION: '1.8.3'
+      conda_python36:
+        DISTRIB: 'conda'
+        PYTHON_VERSION: '3.6'
+      conda_python36_localeC:
+        DISTRIB: 'conda'
+        PYTHON_VERSION: '3.6'
+        LOCALE: 'C'
+      conda_python37_localeC:
+        DISTRIB: 'conda'
+        PYTHON_VERSION: '3.7'
+        LOCALE: 'C'
+        SPHINXOPTS: '-nWT --keep-going'
+      conda_python38_sphinxDev:
+        DISTRIB: 'conda'
+        PYTHON_VERSION: '3.8'
+        SPHINX_VERSION: 'dev'
+      python37_minimal:
+        DISTRIB: 'minimal'
+        PYTHON_VERSION: '3.7'
+      # pythonNightly:
+      #   DISTRIB: 'Nightly'
+  condition: not(contains(variables['Build.SourceVersionMessage'], '[doc only]'))
+
+  steps:
+  - task: UsePythonVersion@0
+    displayName: Specify Python version for minimal
+    inputs:
+      versionSpec: $(PYTHON_VERSION)
+    condition: eq(variables['DISTRIB'], 'minimal')
+  - bash: |
+      export LC_CTYPE=C
+      export LC_ALL=C
+      export LANG=C
+    displayName: Set locale
+    condition: eq(variables['LOCALE'], 'C')
+    # Make sure that things work even if the locale is set to C (which
+    # effectively means ASCII). Some of the input rst files have unicode
+    # characters and we need to deal with this gracefully.
+  - bash: |
+      sudo apt-get install python3-numpy python3-matplotlib python3-pip python3-coverage optipng
+    displayName: Install ubuntu python for ubuntu
+    condition: eq(variables['DISTRIB'], 'ubuntu')
+  - bash: |
+      sudo apt-get install xvfb
+      /sbin/start-stop-daemon --start --quiet --pidfile /tmp/custom_xvfb_99.pid --make-pidfile --background --exec /usr/bin/Xvfb -- :99 -screen 0 1400x900x24 -ac +extension GLX +render -noreset;
```

Comment:
You need to set the `DISPLAY` env var for all runs

https://github.com/mne-tools/mne-python/blob/master/azure-pipelines.yml#L81